templates/user/profile.tmpl

```diff
@@ -16,6 +16,7 @@
 					<div class="content word-break profile-avatar-name">
 						{{if .Owner.FullName}}<span class="header text center">{{.Owner.FullName}}</span>{{end}}
 						<span class="username text center">{{.Owner.Name}}</span>
+						<a href="{{.Owner.Name}}.rss"><i class="ui grey icon tooltip ml-3" data-content="{{.i18n.Tr "rss_feed"}}" data-position="bottom center">{{svg "octicon-rss" 18}}</i></a>
```

Comment:
As above, `.Owner.Name` should be replaced by a link.

Reply:
It appears that `.Owner.Name` is the correct variable to use (see the blue link text at the bottom right):
<img width="1014" alt="Screenshot 2022-04-13 at 6 00 13 PM" src="https://user-images.githubusercontent.com/71804605/163154320-2683af4b-d6de-4b13-b797-afb787e7c048.png">

Reply:
`.Owner.HomeLink`

Reply:
Thanks! Changed.